SonarQube.Common/Serializer.cs

```diff
@@ -17,12 +19,12 @@ namespace SonarQube.Common
     /// </summary>
     public static class Serializer
     {
-        #region Serialisation methods
+        #region Public methods
```

Comment:
Miscellaneous change: refactor slightly to add a new "ToString(model)" method that is useful when testing and debugging.